ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/server/execution/PullQueryExecutorMetricsTest.java

```diff
@@ -112,22 +112,10 @@ public void shouldRecordErrorRate() {
     assertThat(rate, closeTo(0.03, 0.001));
   }
 
-  @Test
-  public void shouldRecordRequestRate() {
-    // Given:
-    pullMetrics.recordRate(3);
-
-    // When:
-    final double rate = getMetricValue("-rate");
-
-    // Then:
-    assertThat(rate, closeTo(0.03, 0.001));
-  }
```

Comment:
Rather than deleting this test, change it to:

```java
@Test	
  public void shouldRecordRequestRate() {	
    // Given:	
    pullMetrics. recordLatency(3000);	
   
    // When:	
    final double rate = getMetricValue("-rate");	

    // Then:	
    assertThat(rate, closeTo(0.03, 0.001));	// <-- numbers need changing. 
  }
```

Otherwise there's no test testing that the rate is updated when `recordLatency` is called!